cli.js

```diff
@@ -6,7 +6,7 @@ program
   .option('-t, --tasks [path]', 'Path to tasks file')
   .option('-b, --single-browser [name]', 'Name for browser that will be used for execution')
   .option('-u, --base-url [url]', 'Base url of website, e.g. http://localhost:8080')
-  .option('-p, --selenium-port [port]', 'Selenium server port')
+  .option('-p, --selenium-port [port]', 'Selenium server port', parseInt)
```

Comment:
Instead of `parseInt` it would be better to use: `function(param) { return parseInt(param, 10); }`. Problem is that validation handler in `option` is called with two params.